BetterMD/elements/table.py

```
@@ -1,113 +1,487 @@
+    def to_rst(self, inner, symbol, parent, **kwargs):
+        content = " ".join([e.to_rst() for e in inner])
+        width = len(max(symbol.table.cols[symbol.header], key=len).data)
+        if content == "":
+            return "".center(width)
+        return f"**{content}**".center(width)
 
-class TdRST(CustomRst):
-    def to_rst(self, inner: list[Symbol], symbol: Symbol, parent: Symbol) -> str:
-        if len(inner) > 1 or not isinstance(inner[0], (Text, H1, H2, H3, H4, H5, H6)):
-            raise TypeError("Table Data may only contain text as inner")
+class ThRST(CustomRst):
```

Ruff (0.8.2)

Redefinition of unused `ThRST` from line 115

(F811)